app/views/ratings/_rating.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= form_with(url: new_rating_path, class: "d-inline-block") do |rating| %>
+    <%= label_tag :rating_value_box, "Your Rating" %>
+    <%= number_field_tag :rating_value, nil, within: 0...6 %>
+
+    <%= hidden_field_tag :burrito_id, burrito.id %>
+    <%= hidden_field_tag :rater_id, @current_user.id %>
```

Comment:
It's more secure to exclude the user id from the form and ignore the `rater_id` entirely in the `params` hash using [strong paramaters](https://uchicago.hosted.panopto.com/Panopto/Pages/Viewer.aspx?id=ada06fe0-0512-4667-9173-ad05013bda49&start=2719.009264). This field may be hidden, but it's not tamper proof. Anyone can open the devtools in their browser and edit the value of a hidden field to be some other value.